packages/jest-cli/src/run_jest.js

```diff
@@ -58,7 +58,9 @@ const getTestPaths = async (
   }
 
   const shouldTestArray = await Promise.all(
-    data.tests.map(test => jestHooks.shouldRunTestSuite(test.path)),
+    data.tests.map(test =>
+      jestHooks.shouldRunTestSuite(test.path, test.context.config),
```

Comment:
How about passing a stripped information about test, e.g. `({path, duration, config})`

Reply:
+1

Reply:
Yeah that seems like the best idea

Reply:
Although duration is not always present, right?